src/Components/DraftFeed/DraftFeed.js

```diff
@@ -0,0 +1,131 @@
+import './DraftFeed.css';
+
+import { Button } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import {
+  DraftablePlayers,
+  players,
+} from './Components/DraftablePlayers';
+import { RobotPlayers } from './Components/DraftablePlayers/RobotPlayers';
+var robotPick = [];
+
+export function DraftFeed() {
+  const [player, setPlayer] = useState([]);
+  const [searchText, setSearchText] = useState('');
+
+  useEffect(() => {
+    fetch('http://localhost:5000/players/', {
+      method: 'GET',
+      headers: {
+        Authorization:
+          'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTE3MTQ5MjksIm5iZiI6MTYxMTcxNDkyOSwianRpIjoiMjg0NjQ1YzAtMzU5OS00ZjJjLWIyYjUtOTNkMGQ2NDllZjUxIiwiZXhwIjoxNjExNzE1ODI5LCJpZGVudGl0eSI6MSwiZnJlc2giOmZhbHNlLCJ0eXBlIjoiYWNjZXNzIn0.M98mjPpObNQLiEf3sUof-cr9QpMHCD5zyNqWkSAGSvU',
+      },
+    }).then((res) =>
+      res.json().then((data) => {
+        setPlayer(data);
+      }),
+    );
+  }, []);
+
+  const handleChange = (event) => {
+    event.preventDefault();
+    setSearchText(event.target.value);
+  };
+
+  const [, setTeam] = useState([]);
+  function addRobot(e) {
+    for (var i = 0; i < 16; i++) {
+      robotPick.push(
+        player[Math.floor(Math.random() * player.length)],
+      );
```

Comment:
do we want the 'other teams' to be assigned players at random? from my understanding the user would be drafting elsewhere and just duplicating the picks into our system. We definitely need a way to set other players as 'taken', but I'm not sure it should be random

Reply:
I agree it should not be random, but this was just a placeholder to show off the UI. The UI was the first user story then the logic was another so this will be altered later when we figure out how we are implementing the draft.